builder/amazon/common/access_config.go

```diff
@@ -60,6 +60,10 @@ type AccessConfig struct {
 	// This allows skipping TLS
 	// verification of the AWS EC2 endpoint. The default is false.
 	InsecureSkipTLSVerify bool `mapstructure:"insecure_skip_tls_verify" required:"false"`
+	// This is the maximum number of times an API call is retried, in the case
+	// where requests are being throttled or experiencing transient failures.
+	// The delay between the subsequent API calls increases exponentially.
+	MaxRetries int `mapstructure:"max_retries" required:"false"`
```

Comment:
What do you think about adding an acceptance test for the config in https://github.com/hashicorp/packer/blob/master/builder/amazon/ebs/builder_acc_test.go